SETUP.md

```diff
@@ -97,7 +97,7 @@ docker run --rm -d \
  http://localhost:18080/  
  user: admin  
  password: admin
-4. To test the  app on a iOS device in your local network, you can use the IP of your machine as the Server URL
+4. To test the  app on a iOS device in your local network, you can use the IP of your machine as the Server URL. To do so, you'll need to add your IP as a trusted domain with a new environment variable `OWNCLOUD_TRUSTED_DOMAINS=192.168.68.64` to the Docker command
```

Comment:
Does it make sense to add that 192.168.68.64 is a sample IP address and needs to be adopted to the own local network?

Reply:
@hosy propose a change? Add "(example)" to make it more clear?

Reply:
When accessing the docker container via localhost:8080 there is no need in setting up the trusted domain env var.

Why is the docker container started via 18080?

Reply:
> When accessing the docker container via localhost:8080 there is no need in setting up the trusted domain env var.

for mobile devices it's required. I guess 18080 is just an example as any other port.

Reply:
> for mobile devices it's required.

why?

Reply:
it's not required if you are in localhost as you stated, so, useless verbosity on my side (sorry!). In case of accessing from mobile, that env variable will add the public ip to the whitelist to access from outside of `localhost` (f. ex, from an external device).